lib/rbUtil.js

```diff
@@ -130,6 +130,35 @@ rbUtil.isTimeUUID = function (s) {
     return uuidRe.test(s);
 };
 
+
+var reqIdBuf = new Buffer(16);
+
+/**
+ * Generates a new request ID
+ */
+rbUtil.generateRequestId = function() {
+    uuid.v4(null, reqIdBuf);
```

Comment:
Actually, a v1 uuid would be nicer, as it also encodes the time. Can change that later though.